lib/nexpose/connection.rb

```diff
@@ -51,14 +51,20 @@ class Connection
     attr_reader :url
     # The token used to login to the NSC
     attr_reader :token
-
     # The last XML request sent by this object, useful for debugging.
     attr_reader :request_xml
     # The last XML response received by this object, useful for debugging.
     attr_reader :response_xml
-
     # The trust store to validate connections against if any
     attr_reader :trust_store
+    # The main HTTP read_timeout value
+    # For more information visit the link below:
+    # https://ruby-doc.org/stdlib-<YOUR_RUBY_VERSION>/libdoc/net/http/rdoc/Net/HTTP.html#read_timeout-attribute-method
+    attr_accessor :timeout
+    # The optional HTTP open_timeout value
+    # For more information visit the link below:
+    # https://ruby-doc.org/stdlib-<YOUR_RUBY_VERSION>/libdoc/net/http/rdoc/Net/HTTP.html#open_timeout-attribute-method
```

Comment:
Can we make this link version-agnostic (i.e. latest) or just pick a version? might be confusing to newbies since it's not a valid link

Reply:
yes